src/batchhafnian.hpp

```diff
@@ -117,6 +136,10 @@ inline std::vector<T> hermite_multidimensional(std::vector<T> &R_mat, std::vecto
                 std::transform(jumpFrom.begin(), jumpFrom.end(), prevJump.begin(), tmpjump.begin(), std::minus<int>());
                 ullint prevCoordinate = vec2index(tmpjump, resolution);
                 H[nextCoordinate] = H[nextCoordinate] - (static_cast<T>(jumpFrom[ii]-1))*static_cast<T>(R(k,ii))*H[prevCoordinate];
+				
+				if (renorm) {
```

Comment:
the renormalization need to occur outside the loop ``` for (ullint jj = 0; jj < Hdim-1; jj++) ``` otherwise you break the recursion relation. So you need a second loop ``` for (ullint jj = 0; jj < Hdim-1; jj++) ``` in which do the renormalization **after** the first loop is completed.

Reply:
Done!